src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -95,6 +95,7 @@ public Claim getClaimByExternalId(final String externalId) {
     @Transactional
     public Claim saveClaim(final String submitterId, final ClaimData claimData, final String authorisation) {
         final String externalId = claimData.getExternalId().toString();
+        final String authToken = userService.generateServiceAuthToken();
```

Comment:
Where is this being used?

Reply:
ignore it, removed now, was just debugging